source/Halibut.TestUtils.CompatBinary.Base/Services/FileTransferService.cs

```diff
@@ -2,7 +2,7 @@
 using System.IO;
 using Octopus.Tentacle.Contracts;
 
-namespace Halibut.TestUtils.Contracts.Tentacle.Services
```

Comment:
The implementation of the service has changed slightly because of the change in data stream, so it has been copied into the backwards compat project

Reply:
Now that we have both versions (one for legacy, and one for regular tests), I wonder if we should rename this to something like `LegacyFileTransferService` to stop confusion. Just a thought.

Reply:
I don't think this is referenced in Halibut tests in a way where the class can be used. So you can never use the wrong one.